asteroid/data/wav.py

```diff
@@ -79,6 +82,47 @@ def data(self):
             self.__wav_data = wav_data
```

Comment:
I know I should have checked sooner but why double underscore here? Instead of single? Double underscore have a special function in Python and I don't see the use of it. Can you explain please?

Reply:
Double underscore marks the variable as   "protected" and makes it harder to change it. Make sense to keep it.